### ISSUE TITLE
Getting an error while trying to Create User

### ISSUE THREAD

**Issue by gjolps** [keephq/keep#2008]
**Describe the bug**
I'm trying to add authentication to keep and when I'm trying to create user from the Keep UI, I get an internal server error. 
I've installed keep using helm and running the latest version.
Below are the logs from keep-backend : 
```
  File "/venv/lib/python3.11/site-packages/keep/api/routes/auth/users.py", line 88, in create_user
    return identity_manager.create_user(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: NoAuthIdentityManager.create_user() got an unexpected keyword argument 'user_email'
```
I also tried to enable database auth by adding ENV vars in values.yml but that did enable authentication. Below are the changes made to the values.yml.
```
backend:
  enabled: true
  env:
    - name: AUTH_TYPE
      value: DB
    - name: KEEP_JWT_SECRET
      value: verysecretkey
    - name: KEEP_DEFAULT_USERNAME
      value: keep
    - name: KEEP_DEFAULT_PASSWORD
      value: keep
frontend:
  enabled: true
  env:
    - name: AUTH_TYPE
      value: DB
```

**To Reproduce**
1. Go to Keep -> Settings -> Users and Access -> Users
2. Click on Add User
3. Enter Email, Name and select Role
4. See error "Error while saving user . An internal server error occurred." 

**Expected behavior**
User needs to be listed under the Users tab.

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [Firefox]
 - Version [130.0.1 (64-bit)]

**Comment by talboren** [comment]
Thank you for reporting, we're looking in to it!

**Comment by talboren** [comment]
@gjolps I have created a pull request (#2009) to address the issue you mentioned here, though I feel like this might not be what you're looking for.
the problem happened because the no_auth (no authentication as the name suggests) identity manager didn't implement add_user correctly (it's pretty much redundant because there are no users when in no_auth mode).

Have you tried reading https://docs.keephq.dev/deployment/authentication/db-auth? After changing to DB auth, you still get the same error?

Join our S[lack community](https://slack.keephq.dev) if you'd like, it'll be easier to commnicate!

**Comment by talboren** [comment]
> Hi @talboren , Thank you for your response. Yes, I made changes to the values.yml file after reading that DB Auth documentation. I added the ENV vars AUTH_TYPE=DB , KEEP_JWT_SECRET=verysecretkey along with default username/password. qq - If I've mentioned AUTH_TYPE=DB, why is it going to no_auth? Did I mention the variable correctly or do I need any modifications there?
> 
> I'm not there on Slack, but I'll make an account post work and join the community by EOD.

Have you changed the env variable in both frontend and backend?

**Comment by gjolps** [comment]
Hi! Yes, I have the variable in both frontend and backend. Earlier, before posting this issue AUTH_TYPE=DB did not redirect me to the sign-in page. But after you merged #2009 , I did a fresh helm install, and it did take me to the login page but I could not login using keep/keep or admin/admin (I added the env for default username and password) . Then I tried changing AUTH_TYPE from DB to SINGLE_TENANT and it worked.
